### PR TITLE
Update dependency

### PR DESCRIPTION
**Pull request by hided62**
Updated dependencies to the latest version

Fixed compilation issues, especially when raising the version of windows crate to 0.48.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
 log = "0.4"
-tokio = { version = "1.8", features = ["net", "sync", "parking_lot", "rt", "io-util", "macros", "process", "fs"] }
 pretty_env_logger = "0.4"
-parking_lot = "0.11"
+parking_lot = "0.12"
 clap = "2.0"
 futures = "0.3"
-windows = "0.17"
+
+[dependencies.tokio]
```
Comment (BusyJay) [inline Cargo.toml:15]: Why not keep the old style?

File: src/ssh.rs [hunk 2]
```diff
@@ -70,38 +69,52 @@ impl Handler {
...
+        let view = match view {
+            Ok(view) => view,
+            Err(_e) => {
+                unsafe {
+                    CloseHandle(handle);
+                }
+                release_token(mask);
+                return Err(other_error(format!(
+                    "failed to map view of file: {}",
+                    Error::last_os_error()
```
Comment (BusyJay) [inline src/ssh.rs]: Why use last os error instead of the error returned by `MapViewOfFile`?

File: Cargo.toml (hunk 1 above)
Comment (hided62) [reply Cargo.toml:15]: I didn't change it for a big reason, mostly because of the line breaks.

File: src/ssh.rs (hunk 2 above)
Comment (hided62) [reply src/ssh.rs]: Right, that's a better way to put it.

File: src/ssh.rs [hunk 3]
```diff
@@ -70,38 +69,52 @@ impl Handler {
     pub async fn new() -> io::Result<Handler> {
         let permit = CONCURRENCY.acquire().await.unwrap();
         let mask = find_available_token();
-        let name = format!("{}-{}\0", FILE_MAP_NAME, mask);
+        let name = HSTRING::from(format!("{}-{}\0", FILE_MAP_NAME, mask));
         let handle = unsafe {
-            CreateFileMappingA(
+            CreateFileMappingW(
```
Comment (BusyJay) [inline src/ssh.rs:74]: Why use the W version instead?

**Comment by BusyJay** [comment]
I just uploaded several commits that updated few dependencies including windows. Thanks for your contribution!

**Pull request closed without merging**